verl/trainer/fsdp_sft_trainer.py

```diff
@@ -394,8 +394,10 @@ def training_step(self, batch: TensorDict):
         n_micro_batches = len(micro_batches)
         step_loss = 0
         for micro_batch in micro_batches:
-            loss = self._compute_loss_and_backward(batch=micro_batch) / n_micro_batches
+            loss = self._compute_loss_and_backward(batch=micro_batch, do_backward=False) / n_micro_batches
+            loss.backward()
```

Comment:
I think we need to move the .backward() with in the context (https://github.com/volcengine/verl/pull/436/files#diff-07c65794985259af3f0f8e156fcf70119ad136cd42e06b4c6584799b1ebc8f2fR300)

otherwise SP will likely cause OOM issue